ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -128,7 +128,7 @@
 {% include 'eos/tcam-profile.j2' %}
 {# Connectivity monitor configuration #}
 {% include 'eos/monitor-connectivity.j2' %}
-{# MAC address-table#}
+{# MAC address-table aging #}
 {% include 'eos/mac-address-table.j2' %}
```

Comment:
we should rename the old config template since you split the area in different templates. (I guess you split it because of ordering in eos)

Reply:
Yes, to maintain the order. I didn't rename it now to get faster approval. I will change it now